script/Ciel/Day 9/09_01_CIEL09_4_1.txt

```diff
@@ -413,37 +451,43 @@ I turn around.
 [sha:6793a3b837c7348a24596533c25f92bd1a0517e5]{
 -- Page 506, Offset 23444.
 -- 「……不良でいいですけど。
-"Just delinquent, if you would...
+"Please, just 'delinquent' is fine...
 }
 [sha:358688a609b64fbf2a9ff1a89f9769b3dd5b70b1]{
 -- Page 506, Offset 23445.
 -- 先輩の家ってこっちの方だったっけ？」
-Wasn't your apartment that way, Senpai?"
+Senpai, remind me, is your apartment this way?"
 }
 [sha:18012fa7935e121942344b99421ead572863165e]{
 -- Page 507, Offset 23446.
 -- 「いえ、まったく違いますね。遠野くん、一度来たことあるでしょう。忘れちゃったんですか？」
-"You've got it completely wrong, Tohno-kun. Didn't you come over once? Have you forgotten already?"
+"No, we're going in completely the wrong direction.
+ Tohno-kun, you've been there once before.
+ Don't tell me you've forgotten already?"
 }
 [sha:38cf23c67e4052a99a8d0ed2c4e49503b278647b]{
 -- Page 507, Offset 23447.
 -- 「覚えてます、忘れません。……ちょっと確認したくなっただけです。もう曲がるところは通り過ぎたなって」
-"No, I remember perfectly well... I just wanted to make sure. I was just thinking that we've already passed the turn you should've taken."
+"No, I remember perfectly well... I only wanted to make sure.
+ It just struck me that we've already passed the turn you should've taken."
 }
 [sha:ab61976504c5551b63fafc740f246c67584d587e]{
 -- Page 508, Offset 23448.
 -- 「はい、正解です。大した記憶力ですね。わたしの家への曲がり道は、ズバリさっき通り過ぎました」
-"Yes, that's right. You have a remarkable memory. We've just now gone past the curved road that leads to my home."
+// Reading ズバリ as decisively -> that ship has long sailed
+"Yes, you're quite right.
+ You have a remarkable memory.
+ We passed the turnoff for my apartment some time ago."
 }
 [sha:e412309c79b43b6adf76787798776fe96900d0ea]{
 -- Page 509, Offset 23449.
 -- そっか、そうだよね、と愛想笑いをする。
-I send her a friendly smile. 'I see. Makes sense.'
+I acknowledge her words with a forced smile.
```

Comment:
I do not think that these silent とs translate well at all. You could maybe do sth like
> Right, of course we did. I manage a forced smile. 

for Shiki but Ciel's just feels incredibly clunky to me if you try and force the そうだよね, and I feel the undertones are extremely apparent even without it.

Reply:
Yeah, it isn't easy, that's for sure.
And most of the time I agree, fragmentary と clauses are better rendered in prose, but in this case, the whole thing just falls apart.

I dunno if I can come up with any better solution than the one I gave, though.

>I force a smile, as if to say 'right, of course'.

>Senpai smiles back at me, as if responding with 'you betcha!'

I suppose something like this would be closest to JP and easier to parse for EN, but it sounds dreadfully descriptive for no reason.

Reply:
I disagree with 'whole thing just falls apart'. The context makes the emotion here pretty clear, and the visuals don't hurt either. I've included both lines (as comments), and we can discuss it with more people when we get here in GPR.

Reply:
Yeah, it's just hard to make it comedic instead of awkward. We'll see if anyone else has any better ideas for phrasing.